src/git_changelog/cli.py

```diff
@@ -99,19 +123,26 @@ def get_parser() -> argparse.ArgumentParser:
         "repository",
         metavar="REPOSITORY",
         nargs="?",
-        default=".",
-        help="The repository path, relative or absolute. Default: %(default)s.",
+        help="The repository path, relative or absolute. Default: current working directory.",
+    )
+
+    parser.add_argument(
+        "--config-file",
+        metavar="PATH",
+        nargs="*",
+        help="Configuration file(s).",
     )
 
     parser.add_argument(
         "-b",
         "--bump-latest",
         action="store_true",
         dest="bump_latest",
-        default=False,
-        help="Guess the new latest version by bumping the previous one based on the set of unreleased commits. "
-        "For example, if a commit contains breaking changes, bump the major number (or the minor number for 0.x versions). "
-        "Else if there are new features, bump the minor number. Else just bump the patch number. Default: %(default)s.",
+        help="""\
+Guess the new latest version by bumping the previous one based on the set of unreleased commits. \
+For example, if a commit contains breaking changes, bump the major number (or the minor number \
+for 0.x versions). Else if there are new features, bump the minor number. Else just bump the \
+patch number. Default: unset (false).""",
```

Comment:
It's a very long string, and IMHO multiline strings are way more readable. But happy to revert, I have no particular preference other than that.

Reply:
Thanks!

If you meant more readable for devs, yes it might be. Though the three lines are each a sentence so that's not terrible to read.

If you meant more readable for users, this text gets wrapped anyway by argparse.

Reply:
I meant for devs.

For users, it is actually a strong use case (by removing the backslashes in the multiline string) because argparse does not wrap (at least it's not doing it on my box)

Reply:
Sorry, I meant it is now wrapping at a reasonable width - it does with the full width of the screen (in my case it is really wide)

Reply:
I see, thanks. Well, that's an issue for another day :slightly_smiling_face: argparse does not give the most fancy output, we can think about how to improve that in another thread :slightly_smiling_face:

Reply:
Rolled back :+1: